nikola/plugins/task/galleries.py

```diff
@@ -308,7 +308,7 @@ def gen_tasks(self):
                 yield utils.apply_filters({
                     'basename': self.name,
                     'name': dst,
-                    'file_dep': file_dep,
+                    'file_dep': file_dep + file_dep_dest,
```

Comment:
It should be enough to just do `+ dest_img_list` here.

Reply:
Yeah, good point